### PR TITLE
GE ISTHMUS/Hyper Sequence

### PR DESCRIPTION
**Pull request by agudmundson**
# GE ISTHMUS/Hyper/HBCD:
Adding functionality to handle GE ISTHMUS/Hyper data. A new custom (__process_hbcd_) function for handling the data has been added to the _ge_pfile.py_. I've also included version 30.x into the _get_hdr_fields.py_.

### CODE REVIEW

**Comment by wtclarke** [comment]
@agudmundson Could you resolve the conflicts? It looks like you were working on an old version of the repo.

File: spec2nii/GE/ge_pfile.py [hunk 1]
```diff
@@ -271,6 +273,143 @@ def _process_gaba(pfile):
...
+    lTE_mask[: 33]   = False                                                                # Remove PRESS
+    lTE_metab        = lTE_metab[:, :, :, :, lTE_mask, :]                                   # Isolated HERCULES
+
+    # Handle Incomplete
+    if lTE_mask.shape[-1] % 4 != 0:                                                         # Incomplete Acquisition
+        old_num_avgs = lTE_mask.shape[-1]                                                   # Old Total Averages
+        new_num_avgs = (lTE_mask.shape[-1] // 4) * 4                                        # New Total Averages
+        lTE_metab    = lTE_metab[:, :, :, :, :new_num_avgs, :]                              # Remove Incomplete
+
+        notestring   = f'{subseq:3d} {subseq_name:<20}'                                     # Note Incomplete Data
```
Comment (wtclarke) [inline spec2nii/GE/ge_pfile.py]: Hi @agudmundson There's these undefined variables `subseq` and `subseq_name`.  What should they be defined as?

**Review by wtclarke — APPROVED** [review]
I think I figured it out and fixed.

File: spec2nii/GE/ge_pfile.py (hunk 1 above)
Comment (agudmundson) [reply spec2nii/GE/ge_pfile.py]: Hi @wtclarke! Ahh yes, that was a bit of code copied over to mirror the Twix implementation. Looks like you updated the string to 80ms HERCULES which is perfect! Did you get everything fixed or or is there anything else I should take a look at?

Comment (wtclarke) [reply spec2nii/GE/ge_pfile.py]: Yes, in this regard, but now a bunch of errors occurring in the tests. I'm very certain that this is unrelated to your changes, and instead related to new package versions. I'll investigate.

**Pull request merged**